### PR TITLE
Add/fix some EE instructions

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: data/languages/eecore.sinc [hunk 1]
```diff
@@ -275,15 +279,25 @@ with : prime=28 {
...
+        hi1 = RSsrc;
+    }
+
+    # 0111 0000 0000 0000 dddd d000 0001 0010
+    :mflo1 RD                                        is RD & rs=0 & rt=0 & zero5=0 & fct=18 {
         RD = lo1;
     }
 
+    # 0111 00ss sss0 0000 0000 0000 0001 0011
+    :mtlo1 RSsrc                                     is RSsrc & rt=0 & rd=0 & zero5=0 & fct=19 {
```
Comment (beardypig) [inline data/languages/eecore.sinc:297]: In the EE instruction set manual this is `fct=17`, I assume the manual is incorrect then?

**Comment by beardypig** [comment]
Thanks for the PR :D Good to see some interest in this, it’s pretty rough around the edges, but hopefully we’ll end up with something that produces a usable output!

File: data/languages/eecore.sinc (hunk 1 above)
Comment (ghost) [reply data/languages/eecore.sinc:297]: Yeah the manual lists the same opcodes for MTHI1 and MTLO1, the values in the instruction table are correct though (section 7.2.1).

Comment (beardypig) [reply data/languages/eecore.sinc:297]: Ah yeah, nice. I like how even in version 6 of a document it’s still wrong :)

**Comment by beardypig** [comment]
Thanks @Eljay, only with team work can we succeed.

**Pull request merged**